kasa/smartcamera/modules/alarm.py

```diff
@@ -0,0 +1,168 @@
+"""Implementation of alarm module."""
+
+from __future__ import annotations
+
+from typing import Literal
+
+from ...feature import Feature
+from ..smartcameramodule import SmartCameraModule
+
+DURATION_MIN = 0
+DURATION_MAX = 6000
+
+VOLUME_MIN = 1
+VOLUME_MAX = 10
+
+
+class Alarm(SmartCameraModule):
+    """Implementation of alarm module."""
+
+    # Needs a different name to avoid clashing with SmartAlarm
+    NAME = "SmartCameraAlarm"
+
+    REQUIRED_COMPONENT = "siren"
+    QUERY_GETTER_NAME = "getSirenStatus"
+    QUERY_MODULE_NAME = "siren"
+
+    def query(self) -> dict:
+        """Query to execute during the update cycle."""
+        q = super().query()
+        q["getSirenConfig"] = {self.QUERY_MODULE_NAME: {}}
+        q["getSirenTypeList"] = {self.QUERY_MODULE_NAME: {}}
+
+        return q
+
+    def _initialize_features(self) -> None:
+        """Initialize features.
+
+        This is implemented as some features depend on device responses.
+        """
+        device = self._device
+        self._add_feature(
+            Feature(
+                device,
+                id="alarm",
+                name="Alarm",
+                container=self,
+                attribute_getter="active",
+                icon="mdi:bell",
+                type=Feature.Type.BinarySensor,
```

Comment:
This should probably also have an explicit category?

Reply:
Done

Reply:
Would info be more relevant, considering this is something a user may want to have active per default on HA?